multi_label_trainer_tfidf.ipynb

```diff
@@ -4,10 +4,9 @@
   "metadata": {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.                keras.Input(shape=(), dtype=tf.string),
</pre><p>Do we need to add <code>keras.Input</code>  in a <code>keras.Sequential</code>  model?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/soumik12345/multi-label-text-classification/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>